bots/models/bot_integration.py

```
@@ -551,6 +551,16 @@ def get_web_widget_config(
             config["apiUrl"] = get_api_route_url(stream_create)
         return config
 
+    def get_web_widget_branding(self) -> dict:
+        return self.web_config_extras.get("branding", {}) | dict(
+            name=self.name,
+            byLine=self.by_line,
+            description=self.descripton,
+            conversationStarters=self.conversation_starters,
+            photoUrl=self.photo_url,
+            websiteUrl=self.website_url,
+        )
```

Pylint (3.3.7)

[refactor] Consider using '{"name": self.name, "byLine": self.by_line, "description": self.descripton, ... }' instead of a call to 'dict'.

(R1735)